cognite_toolkit/cdf_tk/load/_resource_loaders.py

```diff
@@ -1912,76 +1910,3 @@ def count(self, ids: SequenceNotStr[NodeId]) -> int:
     def drop_data(self, ids: SequenceNotStr[NodeId]) -> int:
         # Nodes will be deleted in .delete call.
         return 0
-
-
-@final
-class EdgeLoader(ResourceContainerLoader[EdgeId, EdgeApply, Edge, LoadableEdges, EdgeList]):
```

Comment:
Edges are not used in any module, I think we should remove support for them, and instead add them back if we need them at some point in the future.